### PR TITLE
Proposed updates to Contract Testing

### PR DESCRIPTION
**Pull request by rossmcdonald**
@rowa97 I branched from your PR (#148) with some proposed updates to the Contract Testing article. Let me know what you think.

### CODE REVIEW

File: articles/testing/contract-testing.md [hunk 1]
```diff
@@ -1,37 +1,82 @@
...
+  specification.
+
+- **Governance**: Quickly figure out if an API implementation conforms to the
+  OpenAPI specification that was initially agreed upon. Run a contract test
+  scenario on a schedule to ensure the API never violates the specification.
+
+- **Single Source of Truth**: Your API specification is the single source of
+  truth that describes your API. From it, you can generate documentation, SDKs,
+  mock servers, and more. Incorporating the specification into your testing
+  pipeline ensures that it accurately represents your API implementation
```
Comment (rowa97) [inline articles/testing/contract-testing.md]: what is 'it' referring to? specification or pipeline?

Comment (rossmcdonald) [reply articles/testing/contract-testing.md]: The specification. I'll update this sentence to make it more clear.

File: articles/testing/contract-testing.md [hunk 2]
```diff
@@ -1,37 +1,85 @@
...
+
+3. You will automatically be navigated to the new scenario, complete with
+   contract test assertions for each selected endpoint.
+
+<!-- theme: info -->
+> You will likely need to modify the resulting scenario to fit your use case
 
 ## Automatic Contract Test Assertion
 
-After linking your spec to the Scenario Collection, contract test assertions will be automatically added for step assertions.
```
Comment (rowa97) [inline articles/testing/contract-testing.md]: added for? added to? or added as?

Comment (rossmcdonald) [reply articles/testing/contract-testing.md]: Fixed

**Comment by pytlesk4** [comment]
lgtm

**Pull request merged**